common/lib/capa/capa/customrender.py

```diff
@@ -100,6 +100,25 @@ def get_html(self):
 #-----------------------------------------------------------------------------
 
 
+class DemandhintRenderer(object):
```

Comment:
@cahrens Previously there was single button for all the questions in a problem. And during rendering if `demandhint` was present in the XML then we enable the hint button in [problem.html](https://github.com/edx/edx-platform/blob/master/lms/templates/problem.html), I added this class so that demandhint can render automatically for each problem type.